src/core/quadtree.js

```diff
@@ -0,0 +1,462 @@
+/*jslint vars: true, nomen: true, plusplus: true, continue:true, forin:true */
+/*global Node, QNode */
+
+/*
+
+  Copyright (c) 2014 Michael Sander (speedplane)
+  Based loosely on code written by (c) 2011 Mike Chambers.
```

Comment:
Also, third party code should be placed in the [`/external`](https://github.com/mozilla/pdf.js/tree/master/external) folder.